### PR TITLE
Created aks deploy workflow for the project

### PR DESCRIPTION
**Pull request by temitope777**
WHY: The aks deploy workflow is necessary for automatic deployment
HOW: The aks deploy workflow is embedded in the existing paas workflow

### Trello card
https://trello.com/c/PHx3pfky/482-add-deploy-workflow

### Context
Create deploy workflow for aks 
### Changes proposed in this pull request
embedded the workflow action in existing paas workflow
### Guidance to review
on push to master

### CODE REVIEW

**Comment by temitope777** [comment]
[Add deploy workflow](https://trello.com/c/PHx3pfky/482-add-deploy-workflow)

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -1,7 +1,6 @@
 name: Build and Deploy
 on:
   workflow_dispatch:
-  pull_request:
```
Comment (saliceti) [inline .github/workflows/build.yml:4]: Please add it back

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -522,10 +527,108 @@ jobs:
...
+
+      - name: Trigger Deployment to ${{matrix.environment}}
+        uses: ./.github/workflows/actions/deploy_v2
+        with:
+          environment: ${{matrix.environment}}
+          sha:  ${{ github.sha }}
+          azure-credentials: ${{ secrets.AZURE_CREDENTIALS }}
+          pr:   ${{github.event.number}}
+
+      - name: Determine DfE Sign In Message
```
Comment (saliceti) [inline .github/workflows/build.yml:570]: Please create a trello card to implement DFE sign-in for review apps

File: .github/workflows/build.yml [hunk 3]
```diff
@@ -522,10 +527,108 @@ jobs:
...
+          if_true: ':white_check_mark: DfE sign in route obtained: https://${{env.STATIC_ROUTE}}.london.cloudapps.digital'
+          if_false: ':warning: **DfE sign in route pool exhausted (close some open PRs!)**'
+
+      - name: Post sticky pull request comment
+        if: matrix.environment == 'Review'
+        uses: marocchino/sticky-pull-request-comment@v2
+        with:
+          recreate: true
+          message: |
+                   Review app deployed to https://${{env.REVIEW_APPLICATION}}-${{github.event.number}}.${{env.DOMAIN}}
```
Comment (saliceti) [inline .github/workflows/build.yml:584]: - Create AKS_DOMAIN=test.teacherservices.cloud
- Show AKS review app URL

File: .github/workflows/build.yml [hunk 4]
```diff
@@ -522,10 +529,110 @@ jobs:
...
+    needs: [ prepare ]
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v3
+
+      - name: set-up-environment
+        uses: DFE-Digital/github-actions/set-up-environment@master
+
+      - name: Set environment variable
```
Comment (saliceti) [inline .github/workflows/build.yml:549]: I believe this is not used anywhere, can we remvoe this step?

File: global_config/development.sh [hunk 5]
```diff
@@ -5,3 +5,4 @@ AZURE_SUBSCRIPTION=s189-teacher-services-cloud-test
 AZURE_RESOURCE_PREFIX=s189t01
 KV_PURGE_PROTECTION=false
 PLATFORM=aks
+IMAGE_TAG_PREFIX=sha++++
```
Comment (saliceti) [inline global_config/development.sh:8]: Typo

File: .github/workflows/build.yml (hunk 1 above)
Comment (temitope777) [reply .github/workflows/build.yml]: done

File: .github/workflows/actions/deploy_v2/action.yml [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+  deploy-url:
+    value: ${{ steps.set_env_var.outputs.deploy_url }}
+runs:
+  using: composite
+  steps:
+    - name: Set Environment variables
+      id: set_env_var
+      shell: bash
+      run: |
+        echo  ls terraform
```
[the remark this answers is not in the document]
Comment (temitope777) [reply .github/workflows/actions/deploy_v2/action.yml]: done

File: .github/common_environment.yml [hunk 7]
```diff
@@ -2,9 +2,10 @@
   SLACK_SUCCESS:         '#00ff00'
   SLACK_ICON:            https://raw.githubusercontent.com/DFE-Digital/get-into-teaching-api/master/.github/image.png?size=48
   SLACK_USERNAME:        GiT Workflows
-  SLACK_FOOTER:          School Experience 
+  SLACK_FOOTER:          School Experience
   DOCKER_REPOSITORY:     ghcr.io/dfe-digital/schools-experience
   DOMAIN:                london.cloudapps.digital
+  AKS_DOMAIN:            test.teacherservices.cloud
```
[the remark this answers is not in the document]
Comment (temitope777) [reply .github/common_environment.yml]: done

**Review by saliceti — APPROVED** [review]

**Pull request merged**